ui/src/alerting/components/endpoints/EndpointOverlayContents.tsx

```diff
@@ -111,13 +115,21 @@ const EndpointOverlayContents: FC<Props> = ({
                 minHeight: '43px',
               }}
             >
-              {errorMessage && (
+              {message && (
                 <Alert
-                  color={ComponentColor.Danger}
-                  icon={IconFont.AlertTriangle}
+                  color={
+                    message.remoteState === 'Done'
+                      ? ComponentColor.Success
+                      : ComponentColor.Danger
+                  }
+                  icon={
+                    message.remoteState === 'Done'
+                      ? IconFont.Checkmark
+                      : IconFont.AlertTriangle
+                  }
                   style={{width: 'auto', marginTop: '8px'}}
                 >
-                  {errorMessage}
+                  {message.text}
```

Comment:
What if the `text` is undefined? Should we have a generic message to let the user know that something wrong happened?